### ISSUE TITLE
upgrade issue vers 4.1 to 16.1.1.0

### ISSUE THREAD

**Issue by jmdaru** [DevExpress/AjaxControlToolkit#237]
I've removed all of the references to the old package as far as I can tell, reinstalled it using the Package Manager Console like so:
Install-Package AjaxControlToolkit
I've also installed
HtmlAgilityPack
and 
AjaxControlToolkit.HtmlEditor.Sanitizer

When I run the project I get this lovely guy:
Error	35	Assembly 'AjaxControlToolkit.HtmlEditor.Sanitizer, Version=16.1.1.0, Culture=neutral,  uses 'AjaxControlToolkit, Version=16.1.1.0, Culture=neutral,' which has a higher version than referenced assembly 'AjaxControlToolkit, Version=4.1.50731.0, Culture=neutral, '

**Comment by MikhailTymchukDX** [comment]
Hi @jmdaru ,
Would you please check your `Web.config` for the [`bindingRedirect`](https://msdn.microsoft.com/en-us/library/eftw1fys%28v=vs.110%29.aspx) element presence?
If the element exists, remove it and try to run the project again.

**Comment by jmdaru** [comment]
Thanks for the reply @MikhailTymchukDX . I found the issue late Friday evening though. After upgrading from 4.xxx to 16.1.1.0 all of my references were updated except the one in the bin folder of my project. It showed that it was edited when the newer version was installed, but the file was not replaced. The properties detail on it still reflected the old version. I simply copy/pasta-ed the current version dll into my bin folder to fix the issue.
